### PR TITLE
Add the Istio Component Status into features and FAQs

### PR DESCRIPTION
**Pull request by xeviknal**
Adding the istio component feature in the features list. Also added some questions to the FAQ since I see there have been some issues open lately wrt to that feature.

### CODE REVIEW

File: content/documentation/staging/faq/istio-component-status/add-extra-component.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: How can I add one component to the list
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/add-extra-component.adoc:2]: list --> list?

File: content/documentation/staging/faq/istio-component-status/add-extra-component.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+weight: 4
+---
+:icons: font
+
+If you are interested in adding one more component to the Istio Component Status tooltip, you have the option to add one new component into the https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L484[Kiali CR].
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/add-extra-component.adoc:7]: Not sure a line number is a good idea in a doc link as that yaml changes fairly frequently.

File: content/documentation/staging/faq/istio-component-status/component-not-found.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: One component is Not found but I can see it running. What I can do?
+weight: 2
+---
+:icons: font
+
+The first thing you should do is check into the Kiali CR for the `component_status` field. See the FAQ that shows link:#where-find-config[the specific settings for this feature].
+
+What Kiali does is look for a Deployment which its pods have the `app` label with the specified value in the CR and live in that namespace.
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/component-not-found.adoc:9]: which -> for which
CR and live -> CR, and lives

File: content/documentation/staging/faq/istio-component-status/component-not-found.adoc [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+The `app` label name may be changed form the default (app) and it is specified in this https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L642[field] in the Kiali CR.
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/component-not-found.adoc:10]: form -> from

File: content/documentation/staging/faq/istio-component-status/how-change-component-namespace.adoc [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: How can I change the namespace of a component?
+weight: 3
+---
+:icons: font
+
+There are four different places in the Kiali CR whre you can change the Namespace. See the link:#where-find-config[FAQ] related to this.
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/how-change-component-namespace.adoc:7]: whre you can change -> where you must change

File: content/documentation/staging/faq/istio-component-status/where-find-config.adoc [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: Where I can find the settings for changing the Istio Config Status feature?
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/where-find-config.adoc:2]: Where I can -> Where can I

File: content/documentation/staging/faq/istio-component-status/where-find-config.adoc [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+weight: 1
+---
+:icons: font
+
+There are four different fields in the Kiali CR where you can find the definition of each Istio Component:
+
+- external_services.grafana https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L438[CR section]
+- external_services.istio https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L482[CR section]
+- external_services.prometheus https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L527[CR section]
+- external_services.jaeger https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L566[CR section]
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/where-find-config.adoc:12]: Again, I'm not sure about line # links.  But if you prefer it it's OK by me.

File: content/documentation/staging/faq/istio-component-status/where-find-config.adoc [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title: Where I can find the settings for changing the Istio Config Status feature?
+weight: 1
+---
+:icons: font
+
+There are four different fields in the Kiali CR where you can find the definition of each Istio Component:
```
Comment (jshaughn) [inline content/documentation/staging/faq/istio-component-status/where-find-config.adoc:7]: you can find ->  you will find

File: content/documentation/staging/features/index.adoc [hunk 9]
```diff
@@ -796,6 +796,26 @@ Kiali has different validations to help troubleshoot configurations related to `
 </div>
 ++++
 
+== Istio Component Status
+
+The Istio service mesh architecture is based in different components: from the istiod to Jaeger. Those components are key for the mesh to work well. That is why Kiali runs regular checks looking for the status of these Istio components.
```
Comment (jshaughn) [inline content/documentation/staging/features/index.adoc:801]: Please replace with:
The Istio service mesh architecture is comprised of several components, from istiod to Jaeger.  Each component must work as expected for the mesh to work well overall. Kiali regularly checks the status of each Istio component to ensure the mesh is healthy.

File: content/documentation/staging/features/index.adoc [hunk 10]
```diff
@@ -796,6 +796,26 @@ Kiali has different validations to help troubleshoot configurations related to `
...
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/istio-components-1.24.png" title="Istio components status: components not healthy or found">
+   <img src="/images/documentation/features/istio-components-1.24.png" style="width: 660px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+There are two possible *statuses* which a component may be in: `Not healthy` or `Not found`. The `Not healthy` status means that the deployment doesn't have the desired amount of replicas running. The `Not found` status means that Kiali is not able to find the deployment.
```
Comment (jshaughn) [inline content/documentation/staging/features/index.adoc:813]: There are two possible *statuses* which a component may be in:
-->
A component status will be one of:

Also, a question:  Is there not a healthy status?

File: content/documentation/staging/features/index.adoc [hunk 11]
```diff
@@ -796,6 +796,26 @@ Kiali has different validations to help troubleshoot configurations related to `
...
+
+Regarding the *severity* of each component, there is only to options: `core` or `add-on`. The `core` components are those shown as errors (in red) whereas the `add-ons` are displayed as warnings (in orange).
```
Comment (jshaughn) [inline content/documentation/staging/features/index.adoc:815]: there is only to options:  ->  there are only two options:

**Review by jshaughn — CHANGES_REQUESTED** [review]
A few edits for you...

File: content/documentation/staging/features/index.adoc (hunk 10 above)
Comment (xeviknal) [reply content/documentation/staging/features/index.adoc]: well, technically there is the healthy status. But it is never shown. I've added it to the list.

**Comment by xeviknal** [comment]
@jshaughn comments approached. The content is adapted to yesterday's PR which changes a bit the behavior of this feat. Thanks!

**Review by jshaughn — APPROVED** [review]
LG!

**Pull request merged**